chainguard-dev/edu
# updating existing SBOMs and Attestations doc due to product changes

## Type of change

I realized with the new SBOM features that the existing SBOMs and Attestations doc will be stale. Ville suggested recycling this content to limit the mentions of Enforce and focus on the differences between SBOMs and ATtestations.

To that end, I cut any discussion of how Enforce ingests SBOMs (now covered [here](https://edu.chainguard.dev/chainguard/chainguard-enforce/sboms/sboms-in-enforce/)) and made the article more generic overall.

### What should this PR do?

resolves #842 

### Why are we making this change?


### What are the acceptance criteria? 



### How should this PR be tested?

I'm not 100% certain the Open Source section is the best spot for this doc, but it made sense since there's already an SBOM bucket there. I'm open to moving it. Also, please give the shortened article a read to make sure it still makes sense and is valuable.
